js/stencil-editor.js

```diff
@@ -105,18 +120,39 @@ const StencilEditor = function (container, rand, zones_definition) {
             });
 
         // keyboard events
-        $('#zone_label-' + rand)
-            .on('keypress', function (e) {
+        $(document)
+            .on('keyup', function (e) {
```

Comment:
Listening keyup/keypress events and prevent their default behaviour on the whole document may result in unexpected side effets, for instance for people that uses the keyboard navigation.

Reply:
I've tried to restrict listening only to the container containing the stencil, but sometimes I get strange behavior: I get the impression that sometimes the container loses focus, which has the effect of breaking key shortcuts.

I've fine-tuned shortcut detection so that they are only triggered when the editor is active. So, for example, if a user wants to perform an "escape" and the editor is active, the shortcut will only be caught once by the stencil. If the user wanted to "escape" to another action, he'd just have to do it a second time.

I think this solution is acceptable, as I haven't managed to restrict listening to the container only.